Problem6.py

```diff
@@ -0,0 +1,14 @@
+t = int(input())
+for i in range(1, t+1) :
+    n = list(map(int, input().split()))
+    dict = {}
+    for j in n :
+        if j not in dict :
+            dict[j] = 1
+        else :
+            dict[j] +=1
+    for key, values in dict.items() :
+        if values == 1:
+            print(f"#{i} {key}")
+        elif values == 3:
+            print(f"#{i} {key}")
```

Comment:
```suggestion
        if values == 1 or values == 3:
            print(f"#{i} {key}")
```
`if`와 `elif` 문 내의 수행하는 내용이 같으므로 합쳐서 작성해도 좋을 것 같아요! 😄